apps/avatax/integration/tests/product_without_taxes.spec.ts

```diff
@@ -0,0 +1,176 @@
+import { it } from "vitest";
+import { handler, spec } from "pactum";
+import { string } from "pactum-matchers";
+
+// Trick editor into highlighting gql, wihtout parsing
+const gql = String.raw;
+
+it("should return taxes for product without taxes", async () => {
```

Comment:
should return taxes for product without tax class

Reply:
I'd like us to write better test names. IMO perfect would be if we follow BDD or something similar - so we *really* know what is tested.

We don't have to actually implement BDD. But we can eg:
1. Store BDD scenarios in tools (testmo)
2. Link them from scenario in app code
3. Copy-paster given/when/then pattern to the comment near the scenario

Scenario itself can be longer, but let it be descriptive enough. 
Having nested "describe" can help to organize them

Reply:
I agree that we need better test names. The names should be clear to the 'business people', which are unfamiliar with the feature/test scope. For example, right now `return taxes for product without taxes` doesn't say much to me.

About TestMo please remember that we have only 10 slots and unfortunately we are not going to the higher plan. Of course, we can share them via PDF file.

Reply:
I've updated the test name to use test cases from Testmo and split each individual step in the e2e test into separate `it` so that it has a separate timeout and it's clear on what step the test might have failed